### DISCUSSION TITLE
Does Laragon run on laptops with ARM architecture (Snapdragon CPUs)?

### DISCUSSION THREAD

**Discussion by macurious** [leokhoa/laragon#887] [Q&A]
Does anyone use Laragon on such a device, such as a Microsoft Surface Copilot+ laptop and can confirm that Laragon runs smoothly? I am planning to buy one for travelling, but **need to be sure** that I can work with Laragon. I looked for a compatibility list but couldn't find anything at all. Also, slightly offtopic, what is your opinion on AI-powered laptops for web development? Should I avoid and buy a "normal" laptop instead?

**Comment by adoucette** [comment]
Are there any answers on this yet?

**Reply by AWqxKAWERbXo** [reply]
I didn't read this discussion in 2024 but I will try to answer now: 

It's not about Copilot+ PC's but their architecture: ARM or x86. 

I don't know how smooth Apache and MySQL run through a compatibility layer (x86 to ARM) but it's said that performance on ARM is quite good. 

It's not a Laragon topic but more or less all about Apache and MySQL performance.
